src/tests/server.mjs

```diff
@@ -23,6 +23,11 @@ router.use((request, response, next) => {
   }
 })
 
+router.post("/simple-redirect", (request, response) => {
+  const { path } = request.body
+  response.redirect(303, path)
+})
+
```

Comment:
Existing `/redirect` added  extraneous params to the redirected URL, such as `enctype`.